server/models/SentenceAndOffenceViewModel.ts

```diff
@@ -96,6 +96,14 @@ export default class SentenceAndOffenceViewModel {
       .filter((sentences: PrisonApiOffenderSentenceAndOffences[]) =>
         sentences.some((sent: PrisonApiOffenderSentenceAndOffences) => sent.offences.length > 1)
       )
-      .flatMap(sentences => sentences.map(sentence => [sentence.caseSequence, sentence.lineSequence]))
+      .flatMap(sentences =>
+        sentences.map((sentence: PrisonApiOffenderSentenceAndOffences) => {
+          if (sentence.offences.length > 1) {
+            return [sentence.caseSequence, sentence.lineSequence]
+          }
+          return undefined
+        })
+      )
```

Comment:
the map block can be replaced with ternary operator I think:
`return sentence.offences.length > 1 ? [sentence.caseSequence, sentence.lineSequence] : undefined`

Reply:
Done